feat: relocation of ctype config (#78)
KILTprotocol/web3-login-demo: README.md

Reviewer: Lines 78 to 83 are not up to date anymore.

Author: Done at: [4fe1b4e](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4fe1b4e7e582b11bcf63229d6331fab8e0c7a871)

- `DAPP_DID_URI` = _This is the URI of the Kilt DID that identifies your dApp_
- `DAPP_NAME` = _This should be a custom name for your dApp_
- `JWT_SIGNER_SECRET` = _This is secret key (string) that signs the Json-Web-Tokens before saving them in the Cookies_
- `CTYPE_HASH` = _This is the type of credential (CType) your dApp will request from users for login. If you want to specify more than one CType, you can do so by adding a '/' sign between them_

> - `TRUSTED_ATTESTERS` = _This is a list of attester DIDs (CSV, separated by ','). Only credentials issued by these attesters will be considered valid. If you are using more than one CType, you can specify the trusted attesters by adding a '/' sign between them_

Reviewer: ```suggestion
- `TRUSTED_ATTESTERS` = _This is a list of attester DIDs (CSV, separated by ','). Only credentials issued by these attesters will be considered valid. If you are using more than one CType, indicate the groups of trusted attesters by respectively separating  them  with a '/' sign._
```

Do it similarly for the `REQUIRED_PROPERTIES`

Author: Done at: [4868513](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4868513d625b242ba3307605b8596cfa840d8941)

> - `REQUIRED_PROPERTIES` = _This is a subset of CType properties (CSV, separated by ',') required to be exposed on credential presentation. If you are using more than one CType, you can specify the required properties by adding a '/' sign between them._

Author: Thank yo 😊  really appreciated

### Info about Example/Recommended cTypes

This table provides information on recommended cTypes for SocialKYC credential issuance. SocialKYC is a platform that issues credentials based on user social media presence. The cType-hash is consistent across both Spiritnet and Peregrine to ensure interoperability and consistent verification across the KILT network by using universal identifiers stored on archive nodes.

#### SocialKYC on Spiritnet: did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare

> #### SocialKYC on Peregrine: did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY

Reviewer: The title and the content need to be on different lines. Right now the links on the table do not work.

Reviewer: You can also add the links from the websites here, e.g. https://test.socialkyc.io and  https://socialkyc.io

Author: Done at: [4868513](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4868513d625b242ba3307605b8596cfa840d8941)

- `DAPP_DID_URI` = _This is the URI of the Kilt DID that identifies your dApp_
- `DAPP_NAME` = _This should be a custom name for your dApp_
- `JWT_SIGNER_SECRET` = _This is secret key (string) that signs the Json-Web-Tokens before saving them in the Cookies_
- `CTYPE_HASH` = _This is type of credential (CType) your dApp will request from users for login_

> - `TRUSTED_ATTESTERS` = _This is the DID that identifies the attestors whose attestations are accepted_

Reviewer: ```suggestion
- `TRUSTED_ATTESTERS` = _This is a list of attesters DIDs (c.s.v.`,`). Only credentials issued by them will be considered valid_
```

Author: Done at: [4fe1b4e](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4fe1b4e7e582b11bcf63229d6331fab8e0c7a871)

> - `REQUIRED_PROPERTIES` = _These are the required properties that identify the CType_

Reviewer: ```suggestion
- `REQUIRED_PROPERTIES` = _This is a subset of CType properties (c.s.v.`,`) required to be exposed on credential presentation_
```

Author: Done at: [4fe1b4e](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4fe1b4e7e582b11bcf63229d6331fab8e0c7a871)

> - `REQUIRED_PROPERTIES` = _These are the required properties that identify the CType; it is 'Email' by default_

Author: Fixed at [99ad5a6](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/99ad5a6471d7ebe93ce3eb34c9da8f3e919cc7b3)

To proceed, you may select the desired credential through the CType Environment Variables listed below, allowing you to tailor the authentication process to your needs.

> | Socail KYC | CTYPE_HASH                                                         | TRUSTED_ATTESTERS(spiritnet)                              | TRUSTED_ATTESTERS(peregrine)                              | REQUIRED_PROPERTIES |

Reviewer: The whole table just has some useful information about some recommended cTypes. I would give it an according description and also a **title** (via `###`on markdown). Maybe "Info about example/recommended cTypes" or something like that. 

Please, also signalise somehow that we only recommend SocialKYC and the types of credentials it issues. 
This is also a good place to explain why the cType-hash is the same for both chains.

The first row is not great. The first column should be "Title", not "SocialKYC". 

Both `TRUSTED_ATTESTERS(spiritnet)` and `TRUSTED_ATTESTERS(peregrine)` should rather be `SocialKYC on Spiritnet` or on `SocialKYC on Peregrine`. 

Maybe you can first list the two SocialKYC DIDs and then add references to them on the table. Then you can leave the title Trusted Attesters. That would be more fitting, if later we decide to add an external attester to this list.

You will have to make them subtitles to be able to reference them, like this: 
```
####SocialKYC on the Production: did:kilt:...

And then inside the table (prod. SYKC)[#SocialKYC on the Production]
```

And on the properties column is maybe better to include all properties and let them decide. This would make the "required" part false. But the columns name really don't need to be the name of the CONSTANTS.

Author: Done at: [4fe1b4e](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4fe1b4e7e582b11bcf63229d6331fab8e0c7a871)

| Twitter    | 0x47d04c42bdf7fdd3fc5a194bcaa367b2f4766a6b16ae3df628927656d818f420 | did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare | did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY | Twitter             |
| Discord    | 0xd8c61a235204cb9e3c6acb1898d78880488846a7247d325b833243b46d923abe | did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare | did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY | Discord             |
| GitHub     | 0xad52bd7a8bd8a52e03181a99d2743e00d0a5e96fdc0182626655fcf0c0a776d0 | did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare | did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY | GitHub              |
| Twitch     | 0x568ec5ffd7771c4677a5470771adcdea1ea4d6b566f060dc419ff133a0089d80 | did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare | did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY | Twitch              |
| Telegram   | 0xcef8f3fe5aa7379faea95327942fd77287e1c144e3f53243e55705f11e890a4c | did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare | did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY | Telegram            |

> | Youtube    | 0x329a2a5861ea63c250763e5e4c4d4a18fe4470a31e541365c7fb831e5432b940 | did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare | did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY | Youtube             |

Reviewer: The "YoutubeChanel" CType does not have a property called `Youtube`. 

Please, check the cTypes definitions (maybe on cTypeHub) and list properties that are really inside them. Also please use the official cType title, for example "YoutubeChannel" instead of "Youtube.

Author: Done at: [4fe1b4e](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4fe1b4e7e582b11bcf63229d6331fab8e0c7a871)

Reviewer: Great!
